Pipfile

```diff
@@ -27,6 +27,7 @@ psycopg2-binary = "*"
 bcrypt = "==3.1.7"
 flask-marshmallow = "*"
 marshmallow-sqlalchemy = "*"
+atomicwrites = "*"
```

Comment:
What is this being used for?

Reply:
I wasn't able to run the tests until I manually installed this, since it wasn't in the pipfile

Reply:
hmmm. that seems odd. Perhaps we can move this to the dev/test packages then?

Reply:
Sounds like it's a Windows specific required packages for the testing commands. I've moved it to the dev-packages section